File: matlab_tests/test_conditional.m
% Test if-elseif-else statements
function category = test_conditional(value)
    if value < 0
        category = 'negative';
    elseif value == 0
        category = 'zero';
    elseif value < 10
        category = 'small';
    else
        category = 'large';
    end
end

File: matlab_tests/test_conditional.py
import numpy as np

# Test if-elseif-else statements

def test_conditional(value):
    if (value < 0):
        category = 'negative'
    elif (value == 0):
        category = 'zero'
    elif (value < 10):
        category = 'small'
    else:
        category = 'large'
    
    return category
